ext/standard/iptc.c

```diff
@@ -193,8 +193,8 @@ PHP_FUNCTION(iptcembed)
 	}
 
 	if (iptcdata_len >= SIZE_MAX - sizeof(psheader) - 1025) {
-		php_error_docref(NULL, E_WARNING, "IPTC data too large");
-		RETURN_FALSE;
+		zend_argument_value_error(1, "must be less than %d characters", SIZE_MAX - sizeof(psheader) - 1025);
```

Comment:
I'd go for `is too large` here, the exact value doesn't add value as it's impossible to reach with sensible code.